### DISCUSSION TITLE
archery有增加pg数据库审核功能的计划吗？

### DISCUSSION THREAD

**Discussion by peixubin** [hhyo/Archery#2600] [Q&A]
如果有的话，计划在什么时候发布？

**Comment by LeoQuote** [comment]
可以使用 archery 执行上线工单, 请看这里的逻辑, https://github.com/hhyo/Archery/blob/5f425140615b84f9fe7e5f3549ac41814dad5444/sql/engines/pgsql.py#L292 是带有execute_workflow 方法的, 但是由于没有解析 pg 语法的能力, archery 无法像 mysql 一样对语句进行自动的分析和审核, 这方面也没有相关人员进行贡献, 欢迎你pr.

**Comment by peixubin** [comment]
就是实现execute_check函数吗？

**Reply by LeoQuote** [reply]
参考下 oracle 引擎吧，那边有实现的范例
